src/org/opensolaris/opengrok/configuration/Configuration.java

```diff
@@ -702,8 +769,18 @@ public String getReviewPattern() {
         return reviewPattern;
     }
 
-    public void setReviewPattern(String reviewPattern) {
-        this.reviewPattern = reviewPattern;
+    /**
+     * Set the review pattern to a new value
+     *
+     * @param reviewPattern the new pattern
+     * @throws PatternSyntaxException when the pattern is not a valid regexp or
+     * does not contain at least one capture group
+     */
+    public void setReviewPattern(String reviewPattern) throws PatternSyntaxException {
+        if (!reviewPattern.matches(".*\\([^\\)]*\\).*")) {
+            throw new PatternSyntaxException("The pattern must contain at least one group.", reviewPattern, 0);
```

Comment:
for the application the empty group is enough but it makes change to restrict the user to have at least a single character in it